spec/pluralrules.html

```diff
@@ -64,22 +142,18 @@
 
       <emu-alg aoid="ResolvePlural">
         1. Assert: Type(_pluralRules_) is an Object, and _pluralRules_'s internal slot[[initializedPluralRules]] is *true*.
+        1. Let _nx_ be ? ToNumber(_x_).
+        1. If the _result_ of isFinite(_nx_) is *false*, then
+          1. Return *"other"*.
         1. Let _pluralRuleFunction_ be the value of _pluralRules_.[[pluralRule]].
-        1. If the _result_ of isFinite(_x_) is *false*, then
-          1. If _x_ is *NaN*, then
-            1. Let _n_ be an ILD String value indicating the *NaN* value.
-          1. Else
-            1. If _x_ < 0, then
-              1. let _n_ be -*Infinity*.
-            1. Else
-              1. Let _n_ be *Infinity*.
-        1. Else
-          1. If _x_ < 0, then
-            1. Let _n_ be -_x_.
-          1. Else
-            1. Let _n_ be _x_.
-        1. NOTE n might need to be transform ToString() to preserve decimals.
-        1. Let _result_ be _pluralRuleFunction_(_n_).
+        1. Let _operands_ be _GetOperands_(_x_).
+        1. Let _result_ be _pluralRuleFunction_(
```

Comment:
let's keep it like this for now, I can make the modifications of this line later on.